src/utils/time.js

```diff
@@ -9,7 +9,7 @@ function getTimeUnits(u) {
     'm': '分',
     's': '秒'
   }
-  if (getLangCode() === 'zh') {
+  if (getLangCode(true) === 'zh-hans') {
     return units[u]
   } else {
     return u
```

Comment:
There are no significant discrepancies between the two versions of your code, and there appears to be only minor formatting improvements needed due to some missing whitespaces. The current version of the provided code seems correct according to the knowledge cutoff date you mentioned, although these checks may need adjustments based on updated language standards.

To address any changes that do occur with future developments in programming languages or specific libraries, please consider updating this script accordingly.